pkg/controller/policyrecommendation/policyrecommendation_controller.go

```diff
@@ -65,6 +65,7 @@ func Add(mgr manager.Manager, opts options.AddOptions) error {
 	}
 	licenseAPIReady := &utils.ReadyFlag{}
 	tierWatchReady := &utils.ReadyFlag{}
+	policyRecScopeWatchReady := &utils.ReadyFlag{}
```

Comment:
It seems like we should check this ready flag (like we do with the license and tier ones) before fetching the PolicyRecommendationScopes in the Reconcile.

Reply:
Done, thank you.